tests/kolainst/Makefile

```diff
@@ -22,3 +22,9 @@ install: all
 localinstall: all
 	rm -rf ../kola
 	make install KOLA_TESTDIR=../kola
+
+upgrades-install: all
```

Comment:
Instead of this, we could make it part of the `destructive` suite, but add a e.g. `requiredTag: rpm-ostree-upgrade` to it so that it doesn't run in the default testsuite run, and will only run if you pass `--tag rpm-ostree-reprovision`.

Reply:
SGTM, thanks!